### PR TITLE
Update DE locale

### PR DESCRIPTION
**Pull request by pex**
Currently some tests won’t fix - they are commented out
Inspired by 2c3cd2e1d3cb752dc957c1a7009a59e30f0900cb (SV locale)

### CODE REVIEW

File: src/locale/de/_lib/localize/index.js [hunk 1]
```diff
@@ -1,42 +1,103 @@
...
+  wide: ['vor Christus', 'nach Christus']
+}
+
+var quarterValues = {
+  narrow: ['1', '2', '3', '4'],
+  abbreviated: ['Q1', 'Q2', 'Q3', 'Q4'],
+  wide: ['1. Quartal', '2. Quartal', '3. Quartal', '4. Quartal']
+}
+
+// Note: in English, the names of days of the week and months are capitalized.
```
Comment (leshakoss) [inline src/locale/de/_lib/localize/index.js]: Please change it to `in German`

**Comment by leshakoss** [comment]
@pex why do the commented out tests not work?

**Comment by pex** [comment]
@leshakoss unfortunately I can't tell. All of the test were failing for SV locales, too (see 2c3cd2e1d3cb752dc957c1a7009a59e30f0900cb). Like @alexandernanberg I commented these failing tests.
Parsed date strings result in wrong date objects, though this error doesn't seem connected to the translations itself.

**Comment by kossnocorp** [comment]
Thank you again! The change was released as `2.0.0-alpha.10`. See the change log: https://gist.github.com/kossnocorp/a307a464760b405bb78ef5020a4ab136#v200-alpha10

**Pull request merged**